### PR TITLE
feat: Added button to export to pdf (#2730)

### PR DESCRIPTION
**Pull request by atannus**
- Implements component for an "Export do PDF" button.
- Adds PDF button to SummarySheet.
- Hides buttons on PDF using "hide-print".

### CODE REVIEW

**Comment by Dschoordsch** [comment]
@BartoszJarocki could you please review this?

File: packages/client/modules/email/components/SummaryEmail/ExportToPDF.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+const imageStyle = {
+  verticalAlign: 'middle'
+}
+
+const linkStyle = {
+  height: '40px',
+  padding: '12px 10px 0px 10px',
+  cursor: 'pointer',
+  width: 'fit-content',
+}
```
Comment (BartoszJarocki) [inline packages/client/modules/email/components/SummaryEmail/ExportToPDF.tsx:29]: -1 when working with inline styles, please use the proper type, for example, `const linkStyle: React.CSSProperties`

File: packages/client/modules/email/components/SummaryEmail/ExportToPDF.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+class ExportToPDF extends Component<Props> {
```
Comment (BartoszJarocki) [inline packages/client/modules/email/components/SummaryEmail/ExportToPDF.tsx:31]: -1 could you please convert this to the function component? also, there's no need for this to have any props

File: packages/client/modules/email/components/SummaryEmail/ExportToPDF.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+              <span style={labelStyle}>{label}</span>
+            </div>
+          </td>
+        </tr>
+      </>
+    )
+  }
+}
+
+export default withAtmosphere(withMutationProps(ExportToPDF))
```
Comment (BartoszJarocki) [inline packages/client/modules/email/components/SummaryEmail/ExportToPDF.tsx:57]: -1 no need to use these hoc's

**Review by BartoszJarocki — CHANGES_REQUESTED** [review]

**Comment by Dschoordsch** [comment]
@atannus are you still working on this? Did you want another review round? If so make sure to press the refresh button next to the reviewer who requested changes, otherwise they probably miss it.

**Comment by atannus** [comment]
@Dschoordsch I spoke with @ackernaut a few days ago, I believe the whole PDF endeavor is stalled pending a priorities review by the team.

**Comment by ackernaut** [comment]
Yeah, we don’t want to provide a confusing experience around what’s basically 1) open the print dialog and 2) save as a PDF instead of sending to a printer. We’re not sure we want to recreate this view as a PDF and have 2 views to maintain: 1) the table-based summary 2) a PDF view of it

**Comment by BartoszJarocki** [comment]
@atannus do you mind if I close this PR as it's pretty outdated? once we're back at this issue we can revisit it.

**Comment by BartoszJarocki** [comment]
will do, thank you!

**Pull request closed without merging**